spec/arrays.dd

```diff
@@ -24,11 +24,31 @@ int* p;
 ---------
 )
 
-        $(P These are simple pointers to data, analogous to C pointers.
+        $(P A pointer to type $(I T) has a value which is a reference (address) to another
+        object of type $(I T). It is commonly called a $(I pointer to T).
+        )
+
+        $(P If a pointer contains a $(I null) value, it is not pointing to a valid object.)
+
+        $(P A pointer to $(I T) must either contain a $(I null) value,
+        or point to a valid object of type $(I T).)
```

Comment:
This can be interpreted to mean that even constructing an invalid pointer has undefined behavior, even if it's never dereferenced. I don't think that's the intended meaning.

Maybe: "When a pointer to $(I T) is being dereferenced, it must [...]."

Reply:
ok